src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/CvlProbationSearchRecord.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity.Licence
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.prison.PrisonerSearchPrisoner
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation.CaseloadResult
+
+@Schema(description = "CVL Probation Search Record")
+data class CvlProbationSearchRecord(
+  @Schema(description = "Delius caseload record")
+  val caseloadResult: CaseloadResult,
```

Comment:
mmh.. whats this API representation for? 

(If it's just an intermediate representation then it can be a private inner class in the service)

Reply:
Ah, you've hit the nail on the head there
I thought this was the blanket pattern of creating a representation in Kotlin, I thought it was a bit odd providing schema annotations etc!

I'll refactor it into a private inner class